test/bemhtml-test.js

```diff
@@ -104,4 +105,48 @@ describe('BEMHTML engine tests', function() {
       block: 'b1'
     }, '<div class="b1">ok</div>');
   });
+
+  describe('xhtml option', function() {
+    it('should close short tags in BEMJSON by default', function() {
+      compile('')
+        .apply({ tag: 'br' })
+        .should.equal('<br/>');
+    });
```

Comment:
should be enough to make just 2 `it`'s about default with BEMJSON and `false` with BEMHTML